packages/network-subgraphs/src/operator.ts

```
@@ -142,3 +146,47 @@ export function handleLoss(event: Loss): void {
 //     stake.amount = event.params.amountWei
 //     stake.save()
 // }
+
+export function handleQueuedDataPayout(event: QueuedDataPayout): void {
+    let operatorContractAddress = event.address.toHexString()
+    let dataAmountPT = event.params.amountPoolTokenWei
+    log.info('handleQueuedDataPayout: operatorContractAddress={} blockNumber={} amountDataWei={}', [
+        operatorContractAddress, event.block.number.toString(), dataAmountPT.toString()
+    ])
+
+    let queueEntry = new QueueEntry(operatorContractAddress + "-" + event.params.queueIndex.toString())
+    queueEntry.operator = operatorContractAddress
+    queueEntry.amount = dataAmountPT
+    queueEntry.date = event.block.timestamp
+    queueEntry.delegator = event.params.delegator.toHexString()
+    queueEntry.save()
+}
+
+export function handleQueueUpdated(event: QueueUpdated): void {
+    let operatorContractAddress = event.address.toHexString()
+    log.info('handleQueueUpdated: operatorContractAddress={} blockNumber={}', [
+        operatorContractAddress, event.block.number.toString()
+    ])
+
+    let queueEntry = QueueEntry.load(operatorContractAddress + "-" + event.params.queueIndex.toString())
+    if (queueEntry !== null) {
```

if (queueEntry == null) { throw }

---

or { return } if it's better to ignore the error (maybe log a warning?)